src/power_grid_model_io/config/excel/vision_en.yaml

```diff
@@ -227,27 +288,43 @@ grid:
         - Name
         - ID
     sym_gen:
-      - id: #10
-          - Node.Number
-          - Subnumber
-          - 10
+      - id:
+          auto_id:
+            name:
+              - Transformer loads
+              - generation
```

Comment:
generator?

Reply:
What's it called in Vision? I think we need to keep the names similar

Reply:
Well, we do have the columns named `Generation.P` and `Generation.Q`, so let's keep `generation`.